packages/cache/redis/src/index.ts

```diff
@@ -60,6 +78,7 @@ export default class RedisCache<V = string> implements KeyValueCache<V>, Disposa
           username: parsedUsername,
           password: parsedPassword,
           db: isNaN(numDb) ? undefined : numDb,
+          family: parsedFamily === '6' ? 6 : undefined,
```

Comment:
@ardatan does this needs a separate changeset? seems like a different change?

Reply:
Right! Added!